src/types/apiextensions/v1/conversionreview.ts

```diff
@@ -0,0 +1,29 @@
+import {Resource, Status} from "../../meta";
+
+export declare const apiGroup = "apiextensions.k8s.io/v1";
+
+export declare const conversionReviewKind = "ConversionReview";
+export declare type ConversionReview<TRequest, TResponse> = Resource<
+        typeof conversionReviewKind,
+        typeof apiGroup
+> &
+        (
+                | {
+            request: ConversionRequest<TRequest>;
+        }
+                | {
+            response: ConversionResponse<TResponse>;
+        }
```

Comment:
Since `request` and `response` are mutually exclusive: Do we actually need the two type parameters `TRequest` and `TResponse`, or does a singular type parameter like `TObject` suffice?

Reply:
`TRequest` and `TResponse` actually have to be something different. In the request you get the original object, and in the response you have to provide the new/mapped object